step.yml

```diff
@@ -120,9 +120,9 @@ inputs:
   - app_password: ""
     opts:
       title: "Apple ID: Application-specific password"
-      summary: Required if using Apple ID
+      summary: Optional if using Apple ID
```

Comment:
Note: Might we can better explain this input: required if tfa enabled on the Apple ID.